janusgraph-cassandra/pom.xml

```diff
@@ -103,6 +103,10 @@
                     <groupId>jline</groupId>
                     <artifactId>jline</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>junit</groupId>
+                    <artifactId>junit</artifactId>
+                </exclusion>
```

Comment:
I see this exclusion in
```
/janusgraph-cassandra/pom.xml
/janusgraph-solr/pom.xml
/janusgraph-hadoop-parent/janusgraph-hadoop-2/pom.xml
/janusgraph-hadoop-parent/janusgraph-hadoop-core/pom.xml
```

Is because mrunit, solr, and cassandra are bringing in conflicting version of junit?

Reply:
correct conflicting with the version in junit-vintage